.github/workflows/builds.yml

```diff
@@ -299,8 +299,8 @@ jobs:
           CSC_IDENTITY_AUTO_DISCOVERY: false
           SNAPCRAFT_LOGIN: ${{ secrets.SNAPCRAFT_LOGIN }}
         run: |
-          sudo snap install snapcraft --channel=5.x/stable --classic
-          echo "$SNAPCRAFT_LOGIN" | snapcraft login --with -
+          sudo snap install snapcraft --channel=7.x/stable --classic
+          export SNAPCRAFT_STORE_CREDENTIALS=$(cat "$(SNAPCRAFT_LOGIN)")
```

Comment:
This is not appropriate since `cat` requires a file behind, and `$( SNAPCRAFT_LOGIN )` is trying to execute the command `SNAPCRAFT_LOGIN` which does not exist whereas we only want to get the value in the variable itself. Also you fully removed the login step, which is the important one.